src/utils/setUpRand.m

```diff
@@ -0,0 +1,50 @@
+% (C) Copyright 2010-2020 Sam Schwarzkopf
+% (C) Copyright 2020 CPP_PTB developers
+
+function setUpRand()
+    % setUpRand()
+    %
+    % Resets the seed of the random number generator. Will "adapt" depending on the matlab/octave
+    % version.
+    % It is of great importance to do this before anything else!
+    %
+    % For an alternative from PTB see `ClockRandSeed`
+
+    seed = sum(100 * clock);
+
+    if isOctave
```

Comment:
I think this `if` block can be deleted no?

if we are on octave the series of try catch will eventually hit the correct option line 36, no?

maybe we can just modify the message printed to the screen to say  :

disp('Using Octave or outdated matlab randomizer...');

Reply:
yes good point